### ISSUE TITLE
Extract dependencies

### ISSUE THREAD

**Issue by lucaskasprzyk** [tooploox/KMMSongApp#13]
As we have all dependencies in `module` driven `build.gradle` files it would be good to clean repeated code.
We can start from extracting versions and eventually provide some plugins

**Comment by wzieba** [comment]
Yes, totally agree. Also, I would advise going with `Gradle Extra Properties`, here's why: https://medium.com/@wzieba/an-argument-against-buildsrc-defined-dependencies-or-how-to-seamlessly-keep-your-gradle-8e0bfd7ee8e3?source=friends_link&sk=9f9574c312e72403915a38c88d5fbcea

**Comment by wzieba** [comment]
Also this: https://twitter.com/kotlin/status/1310859700611710978?s=21
